<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-4MKMCSH" failures="2" tests="2" name="TestRunner.TestRunner" time="54.905" errors="0" timestamp="2024-08-04T18:14:56 IST" skipped="0">
  <testcase classname="TestRunner.TestRunner" name="runScenario" time="29.112">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at stepDefinition.SampleTestStepDefs.user_clicks_on_a_button(SampleTestStepDefs.java:39)
at ✽.user clicks on a button(file:///F:/FrameworkSetup/UiAuto/src/Fetures/SampleTest.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="TestRunner.TestRunner" name="runScenario" time="25.793">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at stepDefinition.SampleTestStepDefs.user_clicks_on_a_button(SampleTestStepDefs.java:39)
at ✽.user clicks on a button(file:///F:/FrameworkSetup/UiAuto/src/Fetures/SampleTest.feature:15)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- TestRunner.TestRunner -->
